### PR TITLE
Add attachments and small fix

### PR DESCRIPTION
**Pull request by puiterwijk**

### CODE REVIEW

File: trac_fedmsg_plugin.py [hunk 1]
```diff
@@ -106,6 +112,20 @@ def publish(self, topic, **msg):
...
+    def attachment_added(self, attachment):
+        self.publish(topic='attachment.added',
+                     attachment=attachment2dict(attachment))
+
+    def attachment_deleted(self, attachment):
+        self.publish(topic='attachment.deleted',
+                     attachment=attachment2dict(attachment))
+
+    def attachment_reparented(self, attachment, old_parent_realm, old_parent_id):
+        self.publish(topic='attachment.reparented',
```
Comment (pypingou) [inline trac_fedmsg_plugin.py:124]: reparented? Does this mean updated?

**Comment by pypingou** [comment]
:+1: for d7670bd but for 306e150 we'll should adjust fedmsg_meta before pushing it

File: trac_fedmsg_plugin.py (hunk 1 above)
Comment (puiterwijk) [inline trac_fedmsg_plugin.py:124]: That means that the attachment was moved between parents. So for example from Wiki/Whatever to ticket/3

Comment (pypingou) [inline trac_fedmsg_plugin.py:124]: oh, ok

**Comment by pypingou** [comment]
So what should we do about this one? Split it in two?

**Comment by puiterwijk** [comment]
Since the split hasn't gone into a release yet, I suggest that I will file the meta patch, and then we can merge this after merging that.

**Comment by pypingou** [comment]
wfm

**Comment by bowlofeggs** [comment]
Do we still want this PR, given that infrastructure is retiring Trac?

**Comment by pypingou** [comment]
@puiterwijk did you see @bowlofeggs 's question?

**Pull request closed without merging**